.github/ISSUE_TEMPLATE/Currency_update_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Currency Update Request
+title: "[Currency_Update]: "
+description: "Update request to add support for specific version on ppc64le for available currency package Or fix currency "
+labels: "update currency"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug reports and feature requests; it is
+        not the place for general questions. 
+        Please fill in the following details to request update in Currency for a package:
+  - type: input
+    id: pkg_name
+    attributes:
+      label: Package Name
+      description: >-
+        The package name you want to be supported on power e.g. redis, elasticsearch
+    validations:
+      required: true
+  - type: input
+    id: version 
+    attributes:
+      label: Package Version
+      description: Specific version of package want to be supported on power
+    validations:
+      required: false
+  - type: input
+    id: github_repo
+    attributes:
+      label: GitHub Repo url
+      description: The GitHub source url of package
+    validations:
+      required: true
```

Comment:
Does this also need a "desired packaging" question, e.g. build script or container?  And then, for which version of Linux, e.g. RHEL 9.x, RHEL 8.x, UBI 9?  I don't know if we want to list "all" of the options but that might make it more likely we get requests we are actually willing to handle.

Reply:
we can do that.. and i also had that though in mind.
But as now we are purely focusing on ubi.. also we added that pre-condition in PR to have build script tested in ubi 9 container, not sure if we want to give the option/choice.
open to thoughts @seth-priya . If we want we can add options like we have in dashboard version

Reply:
on question of desired packaging, adding it.
![image](https://github.com/ppc64le/build-scripts/assets/82815888/714bcebe-9047-4900-8d52-d8839741dd21)

Reply:
good on the desired packaging. On, distro, I think, it makes sense to have what you suggested from a portal perspective given what we have today, but @gerrith3  might have additional thoughts, so maybe lets discuss once

Reply:
For the description, how about "Please indicate whether you would like a build script so that you can build the package yourself or a container to run in OpenShift/kubernetes platform, or both."